tests/journald_tests.cpp

```diff
@@ -270,15 +290,30 @@ TEST_F(JournaldLoggerTest, ROOT_LogToJournald)
       statusRunning->executor_id().value(),
       "runs",
       "latest");
-
   ASSERT_TRUE(os::exists(sandboxDirectory));
 
-  string stdoutPath = path::join(sandboxDirectory, "stdout");
-  ASSERT_TRUE(os::exists(stdoutPath));
+  std::string stdoutPath = path::join(sandboxDirectory, "stdout");
+
+  if (GetParam() == "journald") {
+    // Check that the sandbox was *not* written to.
+    // TODO(josephw): This file exists as other parts of the agent will
+    // create it.  We should invert this assertion when we fix this in Mesos.
```

Comment:
I noticed these empty stdout/stderr files in the sandbox. What creates these? Is there a MESOS JIRA for preventing that?